azure-pipelines.vrt-pr.yml

```diff
@@ -15,7 +15,11 @@ variables:
 pool: '1ES-Host-Ubuntu'
 
 jobs:
-  - job: TestVRTool
+  - job: VisualRegressionTest_V9
+    variables:
+      pipelineId: '211'
```

Comment:
Is this an arbitrary `pipelineId` or does it have some connection to an actual pipeline somewhere?

Reply:
Yes this is arbitrary pipelineId. It's an id to identify particular job. Earlier since there was only job we just used the default pipelineid. But since now we have 2 jobs for V8,V9 in same pipeline we created two id's. As long as every job has unique id it should be fine